Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -104,6 +108,18 @@ theorem smul_cancel_of_non_zero_divisor {M R : Type*} [Monoid M] [NonUnitalNonAs
 namespace MulAction
 variable {G α β : Type*} [Group G] [MulAction G α] [MulAction G β]
 
+/-- If a subgroup acts nontrivially, then the type is nontrivial -/
+@[to_additive "If a subgroup acts nontrivially, then the type is nontrivial"]
+theorem isnontrivial_of_nontrivial_action {H : Subgroup G} (h : fixedPoints H α ≠ ⊤) :
```

Comment:
```suggestion
theorem nontrivial_of_fixedPoints_ne_univ {H : Subgroup G} (h : fixedPoints H α ≠ univ) :
```
and can you prove `fixedPoints_of_subsingleton` the contrapositive?

Reply:
I observed that it was not necessary to introduce a `Subgroup`. It is enough to prove it at the level of `G`.